resources/testharness.js

```diff
@@ -20,8 +20,8 @@ policies and contribution forms [3].
     var settings = {
         output:true,
         harness_timeout:{
-            "normal":10000,
-            "long":60000
+            "normal":1000,
```

Comment:
I presume this was for testing, need to revert changes?

Reply:
Certainly.